Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM ibmjava:8-sdk
```

Comment:
removing "IBMJCE" from the list of acceptable providers gives me `java.lang.IllegalStateException: None of the acceptable JCE providers are acceptable: [SunJCE]` when running the tests in this container.